docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -200,8 +202,11 @@ to XFF) are:
   node's connection to Envoy.
 
 In an environment where there are one or more trusted proxies in front of an edge
-Envoy instance, the *xff_num_trusted_hops* configuration option can be used to trust
-additional addresses from XFF:
+Envoy instance, the :ref:`XFF extension <envoy_v3_api_msg_extensions.original_ip_detection.xff.v3.XffConfig>`
+can be configured via the :ref:`original_ip_detection_extensions field
+<envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.original_ip_detection_extensions>`
+to set the *xff_num_trusted_hops* option which controls the number of additional
+addresses that are to be trusted:
```

Comment:
@alyssawilk should the examples below be updated from:

```
    Settings:
      | use_remote_address = true
      | xff_num_trusted_hops = 0
 ```
 
 to:
 
 ```
     Settings:
      | use_remote_address = true
      | original_ip_detection_extensions:
      | - @type: ...
      |    xff_num_trusted_hops = 0
 ```
 
I'd think yes, but maybe too noisy / not needed in this section. WDYT?

Reply:
I think as long as this section somewhere has links to the extension we can just refer to the configurable knob and not include all the gorp